message tab_MissileLevelAttrTable 
{
	optional string tname = 1;
	repeated tab_MissileLevelAttr tlist = 2;
}

message tab_MissileLevelAttr
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 Level = 3;
	optional int32 FollowupTime = 4;
}